From a063dd6dcb761839834fd5d37c50575a7a9a4436 Mon Sep 17 00:00:00 2001
From: Edward Wu <edwardwu@realtek.com>
Date: Mon, 8 Jul 2024 07:23:25 +0800
Subject: [PATCH] media: rtk_vdec_rtd16xxb: Disable VE1 for this stage.

VE1 will use v4l2 stateless in future, and we don't
want to integrate stateful at this moment.
So disable it first.

Change-Id: I892805cad75a17f3f597b1ad10b099fb2d967ea2
---
 drivers/media/platform/rtk_vdec_rtd16xxb/vpu.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/media/platform/rtk_vdec_rtd16xxb/vpu.c b/drivers/media/platform/rtk_vdec_rtd16xxb/vpu.c
index 713ccddb6bc6..5a42d98e4f23 100644
--- a/drivers/media/platform/rtk_vdec_rtd16xxb/vpu.c
+++ b/drivers/media/platform/rtk_vdec_rtd16xxb/vpu.c
@@ -50,6 +50,7 @@ void vpu_cap_buf_done(struct v4l2_fh *fh, struct vb2_v4l2_buffer *buf, bool eos,
 		      enum vb2_buffer_state state);
 
 const static struct vpu_fmt out_fmt[] = {
+#if 0
 	/* video engine VE1 output format */
 	{
 		/* struct v4l2_pix_format */
@@ -215,6 +216,7 @@ const static struct vpu_fmt out_fmt[] = {
 		.misc.bufcnt = 4,
 		.misc.VideoEngine = VIDEO_ENGINE_1,
 	},
+#endif
 	/* video engine VE2 output format */
 	{
 		/* struct v4l2_pix_format */
-- 
2.45.2

